dbbackup/management/commands/mediabackup.py

```diff
@@ -65,12 +65,12 @@ def _explore_storage(self):
     def _create_tar(self, name):
         """Create TAR file."""
         fileobj = utils.create_spooled_temporary_file()
-        tar_file = tarfile.open(name=name, fileobj=fileobj, mode='w:gz') \
-            if self.compress \
-            else tarfile.open(name=name, fileobj=fileobj, mode='w')
+        mode = 'w:gz' if self.compress else 'w'
+        tar_file = tarfile.open(name=name, fileobj=fileobj, mode=mode)
         for media_filename in self._explore_storage():
             tarinfo = tarfile.TarInfo(media_filename)
             media_file = self.media_storage.open(media_filename)
+            tarinfo.size = len(media_file)
```

Comment:
I think this line is wrong:
You define the tar size with the size of only one file (the last)